### ISSUE TITLE
Last value is wrongfully reported

### ISSUE THREAD

**Issue by noldidrita** [software-mansion/react-native-gesture-handler#1023]
I'm having an issue with the PanGestureHandler where the last value is wrongfully reported before reporting the new value. I've also tried LongPressGestureHandler as well, same issue. Let me explain in more detail:
I have a PanGestureRecognizer on a View with two elements (simple Views). I'm allowing the user to drag and move either of the views. I'm using an Animated.event map the y position reported by the handler to an Animated.Value controlling some other (un-related user interactions). I'm noticing that when I pan on the top position of the View everything works fine. Once I release the pan, and try to pan near the bottom of the view, just once, the old Y value is sent to the Animated.event handler (I'm using Animated.debug to print the Y value). Then the correct Y value is reported and everything works fine. As long as I'm panning the same object, it keeps working fine, until I start panning near the top region, where the old value is sent once again. 

So basically, at the exact moment that the gesture handler switches to the Active state, the previous value registered in the handler is sent to the onGestureEvent handler (which is not the correct one). Afterwards everything works fine. I am directly printing the reported y value (and not my animated values) as follows:

```
this._onPanGestureEvent = Animated.event(
      [{
          nativeEvent: {
            y: y =>
              Animated.block([
                Animated.debug('y', y),
                ...
                ])
         }],
      {
        useNativeDriver: true,
      },
    );
```
Let me know if I can provide more information.

**Comment by jakub-gonet** [comment]
This is a bit old, if still relevant, could you please provide some repro?

**Comment by jakub-gonet** [comment]
This issue was not active for some time so I'm closing it.
If you need further help please reply.
